packages/@lwc/engine-server/src/__tests__/fixtures/barrel-lwc-exports/shared.js

```diff
@@ -0,0 +1,11 @@
+export function importsToString(imports) {
+    return imports
+        .map((i) =>
+            i
+                .toString()
+                .split('(')[0]
+                // renderComponent is aliased here: https://github.com/salesforce/lwc/blob/5d01843a7733a03b9ccb59a70ad64af955f15b88/packages/%40lwc/ssr-runtime/src/index.ts#L31
+                .replace('async function serverSideRenderComponent', 'function renderComponent')
+        )
+        .join('\n      ');
+}
```

Comment:
I think we could make this less flaky by using a simple `typeof` instead of stringifying the entire function. I also tend to prefer inlining these kinds of things into fixtures instead of having shared utils (code reuse is not super important for fixture tests).

Reply:
`fn.name` should also work (unless rollup mangles it)

Reply:
Nice, that works thanks